src/kakarot/kakarot.cairo

```diff
@@ -298,6 +299,16 @@ func set_authorized_message_sender{syscall_ptr: felt*, pedersen_ptr: HashBuiltin
     return Kakarot.set_authorized_message_sender(sender, authorized);
 }
 
+@external
+func set_authorized_pre_eip155_tx{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    sender_address: felt, msg_hash: Uint256
+) {
+    Ownable.assert_only_owner();
```

Comment:
If we had a way to send regular starknet tx, we could assert that `tx_info.caller_address == sender_starknet_address` so that pre EIP155 wouldn't be permissioned